src/features/MentorPage/MentorPage.styles.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from 'styled-components';
+
+export const StyledMentorPage = styled.div `
+	background-color: #CDE8F8;
+	position: absolute;
+	width: 100vw;
+	height: 100vh;
+	top: 0;
+	left: 0;
+	display: flex;
+	flex-direction: column;
+`;
+
+export const StyledMentorSearchElement = styled.div`
+  flex: 0 0 auto;
+	max-width: 76vw;
+	background-color: white;
+	border-radius: 10px;
+	margin-left: 12vw;
+	margin-right: 12vw;
+	margin-top: calc(60px + 10vh);
+	height: fit-content;
+`;
+
+export const StyledMentorHeader = styled.div`
+  flex: 1;
+	max-width: 76vw;
+	background-color: #43BFFF;
+	border-radius: 10px;
+	max-height: 80px;
+	height: 80px;
+	color: #1C325D;
+	font-family: "Baloo 2", cursive;
+	font-weight: 700;
+	font-size: 30px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+`;
+
+export const StyledMentorInfoText = styled.div`
+  flex: 0 0 auto;
+	color: #1C325D;
+	font-family: 'Source Sans Pro', cursive;
+	font-weight: 400;
+	font-size: 1.25rem;
+	width: 47%;
+	padding-right: 5%;
+`;
+
+export const StyledMentorSearchDiv = styled.div`
+  flex: 1;
+	display: flex;
+	flex-wrap: wrap;
+	padding: 2rem 0;
+	width: 90%;
+	max-width: 90%;
+	margin: auto;
+`;
+
+export const StyledMentorCardContainer = styled.div`
+	flex: 1;
+`;
```

Comment:
Not a fan of one file that has dozens of components.
As a developer it is easier to check the file-strucure and then see what kind of building blocks the application consists of. Then its nice to see a short definition of the component style. If there is a lot of these - it becomes just a .css file, and then why not use css in the first place

Reply:
Sidenote: I dont think putting _Styled_, in front of the component is needed.

Reply:
I'm under the impression that these would be moved to storybook somehow at some point. How would you do this now that we are waiting for that to be possible?

Reply:
I guess we dont need to put every individual styled div into the Storybook. More like some components that have interactivity in them such as Buttons, Lists, etc.